about.html

```diff
@@ -0,0 +1,108 @@
+             </div>
+            <div class="head">
+             <div class="top-bar">
+                <div class="menu-icon">
+                    <div class="bar"></div>
+                    <div class="bar"></div>
+                    <div class="bar"></div>
+                </div>
+            </div>
+            <h1 class="heading">BATTLE OF THE BANDS TOUR CONCERT 2022</h1>
+            <div class="heading-description">
+                <h4>Explore our school band festivals and tours for concert and jazz bands, wind ensembles, and percussion and marching bands. From clinics and workshops to adjudicated festival competitions to custom concert tours. </h4>
+            </div>
+            <div class="reach-us">
+                <h3>For more information about the battles of the bands contact us through our Email</h3>
+                <h4>battleofbands@gmail.com</h4>
+            </div>
+        </div>
+        </section>
+        <section id="logo">
+            <div class="logo-info">
+                <h3>Battle OF Bands Concerts 2015 Logo</h3>
+                <img src="./assets/Vector.png" alt=""/>
+                <h4>Explore the roots of Africa and experience some of the nation’s finest music in the historic city of Nairobi</h4>
+            </div>
+            <div class="logo-holder">
+                <img src="./assets/bandlogo2.png" alt=""/>
+            </div>
+        </section>
+        <section id="past-battle">
+            <div class="past">
+                <h3>See the past Battle OF Bands Concerts</h3>
+                <img src="./assets/Vector.png" alt=""/>
+                <h4>Take a look ata the last two past battle of the bands concerts in Nairobi Kenya</h4>
+            </div>
+            <div class="past-consert">
+                <div class="past-holder1">
+                    <div class="overlay">
+                        <h3>2019</h3>
+                        <h4>Battle Of Band In Nairobi</h4>
+                    </div>
+                </div>
+                 <div class="past-holder2">
+                   <div class="overlay">
+                        <h3>2021</h3>
+                        <h4>Battle Of Band In Nairobi</h4>
+                    </div>
+                </div>   
+            </div>      
+        </section>
+        <section id="partners">
+            <div class="patner">
+                <img src="./assets/kbc.png" alt="kenya broadcasting"/>
+                <img src="./assets/kaka2.png" alt="kaka empire"/>
+                <img src="./assets/magik.png" alt="magic kenya"/>
+                <img src="./assets/mcsk2.png" alt="MCSK Kenya"/>
+                <img src="./assets/youth2.png" alt="youth kenya"/>
+            </div>
+        </section>
+        <section id="footer-section">
+            <div class="footer">
+            <img src="./assets/wing2.png" alt=""/>
+            <h4>2022 Ultra Music Festival @copyright all rights reserved</h4>
+          </div>
+        </section>
```

Comment:
- [OPTIONAL] Kindly change the `section` tag to the `footer` tag for semantic purposes.
- [OPTIONAL] Kindly change the footer to be dark in the desktop version to match the Behance design.
![image](https://user-images.githubusercontent.com/24703171/163508173-eed924be-18f6-4f1c-a8b7-4fee646231ec.png)